docs/transaction.md

```diff
@@ -0,0 +1,121 @@
+# Weave transactions
+
+_Weave/BNS_ uses protobuf for handling communications
+
+For the beginning, check out the *proto* files and have an idea what are they and how they look like.
+
+## Protocol buffers
+---
```

Comment:
I think section (`##`) is enough for separation, horizontal line `---` is not needed.